app/models/bulkrax/csv_entry.rb

```diff
@@ -351,40 +368,7 @@ def self.matcher_class
       Bulkrax::CsvMatcher
     end
 
-    def collection_identifiers
```

Comment:
@ShanaLMoore is this the method you were talking about? I don't think this is unused. I think it is specifically used in the find method. It gets called the first time and then cached

Reply:
I definitely see it being used in csv_entry.rb. 

```
    def find_collection_ids
      return self.collection_ids if collections_created?
      if collection_identifiers.present?
        collection_identifiers.each do |collection_id|
          c = find_collection(collection_id)
          skip = c.blank? || self.collection_ids.include?(c.id)
          self.collection_ids << c.id unless skip
        end
      end

      self.collection_ids
    end
```